### PR TITLE
Completed Weather JS App

### PR DESCRIPTION
**Pull request by RyelBanfield**
This PR contained the completed weather app project.

Users can search for a city and select the unit they want for their temperature.
Backgrounds change based on the weather.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  const cityTemp = temp;
+
+  return {
+    nameElement, cityWeatherMain, cityTemp,
+  };
+};
+
+// CONSOLE.LOG FUNCTION FOR API
+const consoleLogData = (weatherData) => {
+  console.log(weatherData);
```
Comment (SajjadAhmad14) [inline src/index.js:17]: Please remove `console.log`. It can only be used for debugging.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+};
+
+// WEATHER API HANDLING
+const getWeatherData = async (location, units) => {
+  const apiKey = '303a464ea9aa375660872ba0d5b3afc0';
+  const apiCall = `https://api.openweathermap.org/data/2.5/weather?q=${location}&units=${units}&appid=${apiKey}`;
+  const response = await fetch(apiCall);
+  const data = await response.json();
+  console.log(data);
```
Comment (SajjadAhmad14) [inline src/index.js:26]: Please remove `console.log`. It can only be used for debugging.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      if (keyPressed.which === 13) {
+        const location = document.querySelector('input').value;
+        const unitButtons = document.querySelectorAll('.unit-buttons');
+        if (unitButtons[0].classList.contains('d-none')) {
+          const units = 'imperial';
+          getWeatherData(location, units);
+        } else {
+          const units = 'metric';
+          getWeatherData(location, units);
+        }
```
Comment (SajjadAhmad14) [inline src/index.js:47]: Please remove app logic from DOM-related tasks.

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+const fahrenheitBtn = document.querySelector('.fahrenheit-btn');
+
+celciusBtn.addEventListener('click', () => {
+  if (celciusBtn.classList.contains('d-none')) {
+    celciusBtn.classList.remove('d-none');
+    fahrenheitBtn.classList.add('d-none');
+  } else {
+    celciusBtn.classList.add('d-none');
+    fahrenheitBtn.classList.remove('d-none');
+  }
```
Comment (SajjadAhmad14) [inline src/index.js:80]: Please remove app logic from DOM-related tasks.

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+});
+
+fahrenheitBtn.addEventListener('click', () => {
+  if (fahrenheitBtn.classList.contains('d-none')) {
+    fahrenheitBtn.classList.remove('d-none');
+    celciusBtn.classList.add('d-none');
+  } else {
+    fahrenheitBtn.classList.add('d-none');
+    celciusBtn.classList.remove('d-none');
+  }
```
Comment (SajjadAhmad14) [inline src/index.js:90]: Please remove app logic from DOM-related tasks.

File: src/pushToDom.js [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+  const weatherNameElement = document.createElement('p');
+  weatherNameElement.textContent = weatherMain.main;
+
+  const tempElement = document.createElement('p');
+  tempElement.textContent = Math.floor(temp);
+  if (units === 'metric') {
+    tempElement.innerHTML += '&#8451';
+  } else {
+    tempElement.innerHTML += '&#8457;';
+  }
```
Comment (SajjadAhmad14) [inline src/pushToDom.js:36]: Please remove app logic from DOM-related tasks.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
# Changes Required :recycle:
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hello, you have been doing a tremendous job so far :ok_hand:
## To Highlight 🎉
- [x] You have mastered Git flow.
- [x] PR and commit messages are descriptive.
- [x] You have used ES6 syntax everywhere.
## To Improve 🙏
- [x] Please use the updated linter.yml file as linter config from [here](https://github.com/microverseinc/linters-config/blob/master/javascript/.github/workflows/linters.yml). 
- [x] You have not mentioned the `npm install` command in the `Getting Started` section of your README. The project can not be run locally without running this command. Kindly add it there.
- [x] Toggle functionality from Celsius to Fahrenheit and vice versa is not working. Please make it work. Moreover, when I search for an invalid city name, the app should show a friendly error.
- [ ] async function should handle errors so that it does not crash when an error occurs.
- [x] Please do not use `console.log` because it can slow the app and can make it insecure in the browser.
- [ ] Please do not mix app logic with DOM manipulation tasks.
- [ ] Please see my inline comments for reference.
# Optional
- [ ] It's nice if the app shows the current geolocation of the user when it is first accessed and fetches the weather from that location.
- [ ] Please try to add a new module that only handles weather data fetch. This way your code would be more organized.
### Happy Coding ☺️
## If you have any questions you can comment here in this PR or can always connect with me on slack ( @Sajjad Ahmad ).

File: README.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
+# Todo List
```
Comment (SajjadAhmad14) [inline README.md:1]: The title is wrong. Please fix it!

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: APPROVED✔✔✔**

Hello @RyelBanfield 
Great work making the required changes👏👏

You can merge👌👌👍👍

Happy coding💻💻

**Pull request merged**